<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureCamerasViewPage</class>
 <widget class="QFrame" name="ConfigureCamerasViewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Views</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sandbox cameras</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="DraggableListWidget" name="availableCameraList"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Drag and drop cameras into the view layout.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Views</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="viewsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtBottom</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addViewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Add new cameras view</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/icons/new_indow.png</normaloff>:/images/icons/new_indow.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteViewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Delete the selected  view</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setDefaultViewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Set the selected view as default</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/icons/ok.png</normaloff>:/images/icons/ok.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>View properties</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Cell width:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rows:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="cellWidthSpinBox">
           <property name="minimum">
            <number>160</number>
           </property>
           <property name="maximum">
            <number>2400</number>
           </property>
           <property name="value">
            <number>320</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Cell height:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="cellHeightSpinBox">
           <property name="minimum">
            <number>120</number>
           </property>
           <property name="maximum">
            <number>1800</number>
           </property>
           <property name="value">
            <number>240</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="rowSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="columnsSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Columns:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="viewNameEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>View layout</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="GridWidget" name="gridWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>GridWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DraggableListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">DraggableListWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>availableCameraList</tabstop>
  <tabstop>viewsComboBox</tabstop>
  <tabstop>addViewButton</tabstop>
  <tabstop>deleteViewButton</tabstop>
  <tabstop>setDefaultViewButton</tabstop>
  <tabstop>viewNameEdit</tabstop>
  <tabstop>rowSpinBox</tabstop>
  <tabstop>columnsSpinBox</tabstop>
  <tabstop>cellWidthSpinBox</tabstop>
  <tabstop>cellHeightSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
